metasv/run_spades.py

```diff
@@ -80,7 +80,7 @@ def run_spades_single(intervals=[], bam=None, spades=None, work=None, pad=SPADES
                     spades_log_fd.write("Running spades for interval %s with extraction function %s\n" % (
                     str(interval).strip(), extract_fn_name))
                     retcode = run_cmd(
-                        "bash -c \"timeout %ds %s -1 %s -2 %s -o %s/spades_%s/ -m 4 -t 1 --phred-offset 33 %s\"" % (
+                        "bash -c \"timeout -k 1 %ds %s -1 %s -2 %s -o %s/spades_%s/ -m 4 -t 1 --phred-offset 33 %s\"" % (
```

Comment:
Please remove "-k 1" from this line. We've had users having issues with timeout versions--older versions seem to miss -k option.